bst.py

```diff
@@ -0,0 +1,37 @@
+from tree_node import Node
+
+class BST:
+    def __init__(self) -> None:
+        self.head = None
+    
+    def add_node(self, element):
+        if not self.head:
+            self.head = Node(element)
+            return True
+        
+        temp =  self.head
+        while temp:
+            if element < temp.value:
+                if not temp.left:
+                    temp.left = Node(element)
+                    return True
+                temp = temp.left
+            else:
+                if not temp.right:
+                    temp.right = Node(element)
+                    return True
+                temp = temp.right
```

Comment:
Write a note on how you will handle duplicate values

Reply:
Note can be a comment or docstring, it needn't be code, an understanding/approach of how you'll resolve it not actual code!

Reply:
Done